drivers/unix/os_unix.cpp

```diff
@@ -170,6 +164,7 @@ Error OS_Unix::get_entropy(uint8_t *r_buffer, int p_bytes) {
 		left -= chunk;
 		ofs += chunk;
 	} while (left > 0);
+// Define this yourself if you don't want to fall back to /dev/urandom.
 #elif !defined(NO_URANDOM)
```

Comment:
@Faless I don't know if there's much use for making this opt-out. Did you have specific use cases in mind where some users might not have `UNIX_GET_ENTROPY`, not want to use `/dev/urandom`, and would not be writing their own platform port overriding `OS_Unix::get_entropy`?